.github/workflows/deploy-dev.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         run: |
           cd vue-frontend
           echo VITE_RECAPTCHA_SITE_KEY=${{ secrets.RECAPTCHA_SITE_KEY }} >> .env
+          echo VITE_MIX_PANEL_TOKEN=${{ secrets.MIX_PANEL_TOKEN }} >> .env
```

Comment:
For dev, let's use empty string here instead of real token. Also, as this is not a secret token, maybe we can use add this in config.